tests/fertiscan/inspection_export.json

```diff
@@ -1,5 +1,6 @@
 {
     "inspection_id": "61c93c2f-78ae-4f90-be3c-fc888087bfb9",
+    "inspection_comment": "",
```

Comment:
What's the difference between inspection.json and inspection_export.json? Is it intended that one has `inspection_comment` and not the other?